control/velocity_controller_lqr/velocity_controller_lqr/velocity_controller_lqr_lib.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
+import math as math
+
+import numpy as np
+
+import control as ct
+
+
+class LQRController:
+    def __init__(
+        self,
+        q_surge,
+        q_pitch,
+        q_yaw,
+        r_surge,
+        r_pitch,
+        r_yaw,
+        i_surge,
+        i_pitch,
+        i_yaw,
+        max_force,
+    ):
```

Comment:
This is too many inputs. Either make a dataclass with all of those and have the dataclass as the input, or initialize them to zero in __init__ since you will overwrite them in the ros node either way.

Reply:
If you go for the second option, you can have another method which initializes all the variables to zero and then just call that method in __init__